src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1130,6 +1130,21 @@
      * The statement terminator ({@code ;}).  Depending on the
      * context, this make occur as a sibling, a child, or not at all.
      *
+     * <p>For example:</p>
+     * <pre>boolean var = true;</pre>
```

Comment:
This example is too big. I suggest to use something like
```java
       for (;;)
        ;
```
instead.

Reply:
@pbludov done